core/client/app/controllers/settings/tags.js

```diff
@@ -31,13 +27,7 @@ export default Ember.Controller.extend({
     }),
 
     actions: {
-        enteredMobile: function () {
-            this.set('isMobile', true);
-        },
-
         leftMobile: function () {
```

Comment:
I am rather doubtful the contents of this function are doing anything. As far as I can tell the route for tags is always on the first tag anyway. Perhaps it is legacy behavior? It would be nice to get rid of since it would allow us to drop the observer in `gh-tags-management-container.js`